rust/datafusion/src/logicalplan.rs

```diff
@@ -713,6 +718,9 @@ impl fmt::Debug for Expr {
         match self {
             Expr::Alias(expr, alias) => write!(f, "{:?} AS {}", expr, alias),
             Expr::Column(name) => write!(f, "#{}", name),
+            Expr::ScalarVariable(variable_names) => {
+                write!(f, "#{}", variable_names.join("."))
```

Comment:
this naming is already being used by columns (one line above). Doesn't this make it confusing?

Reply:
Yes, i remove the code '#', the variable name itself start with an @ sign